### DISCUSSION TITLE
Can't access Github Copilot with Student Discount

### DISCUSSION THREAD

**Discussion by shubhambhatnag** [community/community#41059] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello, I used to be in the technical preview for copilot, and have just learned that it remained free for users with a Student coupon. I was approved for one in 2019, and it appears to still be in effect. However, when I go to the copilot section in Settings, it only suggests that I start a free trial.

Free trial image: https://imgur.com/a/z6glDEx 
Approval proof: https://imgur.com/a/ZhDGBGz

I'd appreciate any help!

**Comment by iamcjwilliams** [comment]
@BlobKnight Sorry for the trouble here, please visit [here](https://education.github.com/discount_requests/pack_application) in order to signup for free student access.
